docs/_tutorials/getting_started.md

```diff
@@ -16,6 +16,8 @@ This guide is meant to walk you through getting up and running with a Slack app
 
 When you're finished, you'll have this ⚡️[Getting Started with Slack app](https://github.com/slackapi/bolt-python/tree/main/examples/getting_started) to run, modify, and make your own.
 
+> 💡 For this guide, we are going to be using [Socket Mode](https://api.slack.com/apis/connections/socket), our recommended option for those just getting started and building something for their team. If you already know you're going to want to use HTTP as your app's communication protocol, head over to our parallel guide, [Getting Started over HTTP](/bolt-python/tutorial/getting-started-http). 
```

Comment:
If we have this notice **at the top of the page** (I know we have the section in the middle in the bolt-js one), we may want to skip the steps to set `SLACK_SIGNING_SECRET` env variable and spin up a web server with 3000 port. With this, developers can directly turn Socket Mode on right after new app creation. Thoughts?

Reply:
@seratch  Yeah, with this notice at the top, I think it's fair to say that it then becomes a little more misleading when we do initial setup with `SLACK_SIGNING_SECRET`

* Option 1: We keep this disclaimer at the top, and then we include the app-level-token setup along with the xoxb token setup. 
* Option 2: We move the disclaimer to the middle (in "Setting up events") as it is in the bolt-js guide and then keep the `SLACK_SIGNING_SECRET` steps.

Option 1 is potentially less misleading, but one possible downside  is there's a bit more concepts to introduce before devs can even start the app. 

Any strong preference?

Reply:
My preference is option 1 as it's much simpler and easier for many. That being said, it's not a so much strong opinion. Either way, bolt-js and bolt-python should be consistent here (I don't say bolt-java should be the same too as it has been a bit different from the beginning).

Reply:
Thanks for that feedback @seratch !

I've pushed up some changes in line with taking Option 1. It's simpler to follow since all setup steps are now in the same section ( i.e. getting `xapp` is now in the same section as `xoxb` and toggling on Socket Mode) so there'll be less switching around from code to app config for the dev. And, omitting the signing secret part from code samples means we avoid introducing a confusing concept that's not useful ultimately unless you want to use Request URL. 

Since there's now a note at the top and in the middle AND the bottom with links to get to the standalone guide for Getting Started with HTTP, I'm satisfied there's ample departure points for people with context on why they might want to switch. Let me know what you think!

Reply:
@srajiang I also like the revised version a lot! Great work 👍 

>Since there's now a note at the top and in the middle AND the bottom with links to get to the standalone guide for Getting Started with HTTP, I'm satisfied there's ample departure points for people with context on why they might want to switch. Let me know what you think!

When I read it as a whole, I felt the same! I'm sure that the current version does not have any confusing part for both Socket Mode users and HTTP mode users.